cli/README.md

```diff
@@ -0,0 +1,72 @@
+# Subsembly CLI
+
+`subsembly` is a cli tool that is used to ease the devopment of Subsembly runtimes.
+
+## Installation
+
+To install `subsembly`, run:
+
+`npm install -g subsembly`
+
+## Commands
+
+### `subsembly`
+
+```
+# Default command, displays help window
+subsembly 
+
+# Help command (alternatively, --help)
+subsembly -h
+```
+
+### `subsembly compile`
+Compiles current Subsembly project. Specifically, installs dependencies, generates `metadata` and `dispatcher` files and compiles the Subsembly runtime into hex encoded wasm binary file.
```

Comment:
Users dont know what `metadata` and `dispatcher` files are. We could better describe the command as:
```
Compiles current Subsembly project. More specifically it installs the dependencies and generates the required files for the Runtime to support the `metadata` interface used by PolkadotJS. Lastly, it compiles the Subsembly Runtime into the hex encoded wasm binary file (WASM blob)
```